### ISSUE TITLE
Shouldn't Playwright's helper `fillField` implementation use the `fill` API?

### ISSUE THREAD

**Issue by wadamek65** [codeceptjs/CodeceptJS#3158]
### Description
For context, we have ran into a few situations when calling `fillField` or `clearField` with Playwright have produced results that should not be possible from the user's perspective. For example, in a react application, we had a field and change state when calling `clearField` on an empty input which seemed to produce states that should not be achievable via direct user interaction. This was probably caused due to the fact that `type` also triggers key events as described here: https://playwright.dev/docs/api/class-locator/#locator-fill

I took a look [at the implementation](https://github.com/codeceptjs/CodeceptJS/blob/2fd3fe4bd6e9fd183df6d72396f6bb143a8dc018/lib/helper/Playwright.js#L1325), and it seems like the underlying `fillField` implementation is calling the Playwright's `type` API with an empty string. This prompted two questions:
1. Does this implementation make sense? Isn't this an interaction (typing an empty string) that will never be possible from a user's perspective?
2. Wouldn't it be better to call the `fill` API instead which also checks for actionability? https://playwright.dev/docs/api/class-locator/#locator-fill

**Comment by DavertMik** [comment]
Yes, these are valid points. Our implementation can be outdated. Could you make a pull request with the change?

**Comment by wadamek65** [comment]
@DavertMik If what I wrote in my initial comment makes sense and we would like to indeed adjust it then probably yes. I will try to get to it in the nearest future.

**Comment by nicostubi** [comment]
Hi @DavertMik 

Do you still have this idea to male a pull request with the change describe above?

I have some random issues with the .fillField implementation (timeouts, not waiting for the text to be effectively written, transient error), and this issue #3158 makes me think I could benefit from this very improvement.

BR
Nicolas

**Comment by mirao** [comment]
Maybe [3.5.4](https://codecept.io/changelog/#_3-5-4) solved it by

> Move from ElementHandle to Locator.

?

**Comment by kobenguyent** [comment]
Closed for now! Feel free to reopen if you still encounter the issue with latest version and would be nice to provide a sample code to reproduce the issue.
